auth0_flutter_platform_interface/lib/src/web/client_options.dart

```diff
@@ -0,0 +1,111 @@
+import '../../auth0_flutter_platform_interface.dart';
+
+/// The options for the underlying client on the web platform.
+class ClientOptions {
+  /// The account to use, including the `domain` and `clientId` values.
+  final Account account;
+
+  /// A maximum number of seconds to wait before declaring background calls to
+  /// `/authorize` as failed for timeout. Defaults to 60 seconds.
+  final int? authorizeTimeoutInSeconds;
+
+  /// The location to use when storing cache data. Valid values are `memory` or
+  /// `localstorage`. Defaults to `memory`.
+  final CacheLocation? cacheLocation;
+
+  /// The domain the cookie is accessible from.
+  ///
+  /// If not set, the cookie is scoped to the current domain, including the
+  /// subdomain.
+  ///
+  /// Note: setting this incorrectly may cause silent authentication to stop
+  /// working on page load.
+  ///
+  /// To keep a user logged in across multiple subdomains set this to your
+  /// top-level domain and prefixed with a `.` (for instance: `.example.com`).
+  final String? cookieDomain;
+
+  /// The timeout for HTTP calls using fetch.
+  ///
+  /// Defaults to 10 seconds.
+  final int? httpTimeoutInSeconds;
+
+  /// Whether an additional cookie with no SameSite attribute is set.
+  ///
+  /// This is used to support legacy browsers that are not compatible with the
+  /// latest SameSite changes.
+  ///
+  /// This will log a warning on modern browsers, you can disable the
+  /// warning by setting this to `false` but be aware that some older
+  /// user agents will not work.
+  ///
+  /// See https://www.chromium.org/updates/same-site/incompatible-clients
+  ///
+  /// Defaults to `true`.
+  final bool? legacySameSiteCookie;
+
+  /// The number of days until the cookie `auth0.is.authenticated` will expire.
+  ///
+  /// Defaults to `1`.
+  final int? sessionCheckExpiryDays;
```

Comment:
Given we already have a `httpTimeoutInSeconds`, should this one be `sessionCheckExpiryInDays`?

Reply:
(same for the `onLoad` param)